de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/editor/internal/EditorAPI.java

```diff
@@ -418,13 +419,15 @@ private static IWorkbenchWindow getActiveWindow() {
     }
 
     /**
+     * @param refernencePointManager
+     *            TODO
      * @return the path of the file the given editor is displaying or null if
      *         the given editor is not showing a file or the file is not
      *         referenced via a path in the project.
      */
-    public static SPath getEditorPath(IEditorPart editorPart) {
+    public static SPath getEditorPath(IEditorPart editorPart,
+        IReferencePointManager refernencePointManager) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused method parameter "refernencePointManager". ([squid:S1172](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1172))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=274&issues=AWaXS6DVYAGQNfrBSSyZ&open=AWaXS6DVYAGQNfrBSSyZ)

[ISSUE_KEY]: AWaXS6DVYAGQNfrBSSyZ